package.json

```diff
@@ -30,7 +30,8 @@
     "tokens": "yarn workspace @kiwicom/orbit-design-tokens",
     "tracking": "yarn workspace @kiwicom/orbit-tracking",
     "update-supported-browsers": "markdown --path .github/contribution/testing-conventions.md && git add .github/contribution/testing-conventions.md",
-    "docker:reset": "rm -rf packages/orbit-components/playwright/.cache && yarn install --frozen-lockfile --ignore-scripts"
+    "docker:reset": "rm -rf packages/orbit-components/playwright/.cache && yarn install --frozen-lockfile --ignore-scripts",
+    "deploy:report": "yarn surge playwright-report/"
```

Comment:
Couple of questions to try to understand better the workflow:
  1) Is the `playwright-report/` folder generated automatically by `playwright merge-reports`?
  2) Is the `all-blob-reports/` inside this folder?

Reply:
1. [Yes](https://playwright.dev/docs/test-sharding#:~:text=This%20will%20produce%20a%20standard%20HTML%20report%20into%20playwright%2Dreport%20directory.)
2. No. Inside this folder there will be an `index.html` file and a `data/` folder that are ready to be published on the web. The `all-blob-reports/` folder is generated by the download-artifacts action that downloads the 4 artifacts and merges them all on that single folder. The `playwright merge-reports` expects that folders with multiple blobs inside to merge them all in one.